ansible/roles/oso_console_extensions/files/console-extensions-template.yml

```diff
@@ -0,0 +1,139 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: online-console-extensions
+
+parameters:
+- name: NAME
+  description: The name of the DeploymentConfig
+  value: online-console-extensions
+- name: GIT_REPO
+  description: Git repository housing the code to build and deploy
+  value: "https://github.com/openshift/online-console-extensions.git"
+- name: GIT_REF
+  description: Git branch, commit, or tag to deploy
+  value: "master"
+- name: TARGET
+  description: One of free|paid
+  value: "free"
+
+objects:
+- kind: BuildConfig
+  apiVersion: v1
+  metadata:
+    name: ${NAME}
+    labels:
+      app: ${NAME}
+  spec:
+    source:
+      git:
+        uri: ${GIT_REPO}
+        ref: ${GIT_REF}
+    strategy:
+      sourceStrategy:
+        env:
+        - name: TARGET
+          value: ${TARGET}
+        from:
+          kind: DockerImage
+          name: ${NAME}-builder:latest
```

Comment:
For now, to test this, I've created this builder image in the devenv.

Reply:
EDIT: I've added the building of the custom-builder into the devenv playbook, so to test, run the online devenv playbook.